docs/getting-started/0-agent.md

```diff
@@ -28,6 +31,185 @@ const agentConfig: InitConfig = {
 const agent = new Agent(agentConfig, agentDependencies)
 ```
 
+## Complete Agent Initialization
+
+This is the optimal initialization code for a scenario where complete functionality is needed.
+We will walk through the following steps to initialize the agent with full capabilities.
+
+### 1 - Import statements
+
+```ts
+import {
+  Agent,
+  AutoAcceptCredential,
+  ConnectionEventTypes,
+  ConnectionInvitationMessage,
+  ConnectionRecord,
+  ConnectionStateChangedEvent,
+  ConsoleLogger,
+  CredentialEventTypes,
+  CredentialRecord,
+  CredentialState,
+  CredentialStateChangedEvent,
+  HttpOutboundTransport,
+  InitConfig,
+  LogLevel,
+} from '@aries-framework/core'
+import { agentDependencies } from '@aries-framework/react-native'
+```
+
+### 2- Download genesis file (Optional)
+
+####Mobile agent context:
+When working with an Indy network, the ledger’s genesis file contains the information necessary for an agent to connect to that ledger. In other words, the genesis file contains the schema (like a DB schema) of the target Indy node ledger you are attempting to connect to.
```

Comment:
The genesis file is not really a schema like a generic DB schema. It is more the initial state of your blockchain. eg. block zero.

Reply:
done